docs/instance.md

```diff
@@ -11,16 +21,42 @@ An instance is created on the command line:
 ```sh
 $ cozy-stack instances add <domain>
 ```
+With some possible additional options
+
+- `--locale <lang>`
+- `--email <email>`
+- `--environment <dev/test/production>`
+- `--apps <app1,app2,app3>`
+- `--home <cozy-home>`
+- `--onboarding <cozy-onboarding>`
+- `--registry https://registry.cozycloud.cc`
+
+It registers the instance in a global couchdb database `global/instances`
+```json
+{
+    "hostname": "example.cozycloud.cc",
+    "dbprefix": "example-clozycloud-cc/",
+    "fsroot": "/usr/lib/cozy/example.cozycloud.cc/fs/"
```

Comment:
`/var/lib` would be better than `/usr/lib`. It can be sometimes a separate partition, with more disk space.